glib/connect.go

```diff
@@ -83,7 +79,10 @@ func (v *Object) ConnectAfter(detailedSignal string, f interface{}) SignalHandle
 // closure's first argument to ensure that it is correct, otherwise it will
 // panic with a message warning about the possible circular references. The
 // receiver in this case is most often the first argument of the callback.
-var ClosureCheckReceiver = true
+//
+// This constant can be changed by using go.mod's replace directive for
+// debugging purposes.
+const ClosureCheckReceiver = false
```

Comment:
@diamondburned Can you explain how to use a `go.mod` replace directive to change this constant?

Reply:
You can fork the repository or clone it down, then use the `replace` directive inside `go.mod`, like so:

```
# cloned
replace github.com/gotk3/gotk3 => ../path/to/gotk3
# forked
replace github.com/gotk3/gotk3 => github.com/username/gotk3 commithash...
```